helpers/helpers.go

```diff
@@ -400,18 +400,24 @@ func CheckTokenExpiration(accessToken string) error {
 	switch untilExp := time.Until(expiration); {
 	case untilExp < 0:
 		return fmt.Errorf("the provided access token has expired, please renew it")
-	case untilExp > 0 && untilExp < 24*time.Hour:
-		fmt.Fprintln(
+	case untilExp >= 0 && untilExp < 2*time.Hour:
```

Comment:
```suggestion
	case untilExp >= 1*time.Minute && untilExp < 2*time.Hour:
```

If time remaining is 0 the token is essentially expired.

Reply:
good point, done